src/tailwind.css

```
@@ -0,0 +1,3 @@
+@import "tailwindcss";
+
+@source "./src";
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)